pages/dkp/kaptain/2.0.0/install/uninstall/index.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: layout.pug
+navigationTitle: Uninstall Kaptain
+title: Uninstall Kaptain
+menuWeight: 80
+excerpt: Uninstall Kaptain
+beta: false
+enterprise: false
+---
+
+## Uninstall Kaptain
```

Comment:
Note: 
If installed with helm, we need a command (maybe not desired path) 
If installed with appdeployment, uninstall via UI (disable)